CoreUtilities/src/au/gov/asd/tac/constellation/utilities/gui/JNumberedTextArea.java

```diff
@@ -38,6 +38,10 @@ public void updateLineNumbers() {
         setText(getLineNumbersText());
     }
 
+    protected String getSetLineNumbersText() {
+        return getText();
```

Comment:
I don't think this function is needed as `getText()` is public (and the name is a little confusing)

Reply:
Absolutely! Removed now.